compiler/ghc/default.nix

```diff
@@ -326,4 +332,6 @@ in stdenv.mkDerivation (rec {
   dontStrip = true;
   dontPatchELF = true;
   noAuditTmpdir = true;
-})
\ No newline at end of file
+});
+
+in drv
```

Comment:
What’s up with this?

Reply:
I needed to pass drv to envDeps and exactDeps above